src/main/java/com/labsynch/labseer/dto/LotsByProjectDTO.java

```diff
@@ -13,21 +13,30 @@ public class LotsByProjectDTO {
     private Date registrationDate;
     private String parentCorpName;
     private String project;
+    private String chemist;
+    private String vendor;
+    private String supplier;
 
     public LotsByProjectDTO(
             long id,
             String lotCorpName,
             int lotNumber,
             Date registrationDate,
             String parentCorpName,
-            String project) {
+            String project,
+            String chemist,
+            String vendor,
```

Comment:
I think we should be specific that this is the vendorName not just vendor.  I also think we should include the vendor.code_name as "vendorCode"

@bffrost What do you think?

Reply:
I agree with what @brianbolt wrote. Let's have `vendorName` and `vendorCode`. I think in most cases those are the same value but I can't imagine it'll hurt performance to have an extra field from that same table, so seems better to have more than to have less

Reply:
Updated with the changes, tested the route and it looks good:

```
[
  {
    "chemist": "bob",
    "id": 55,
    "lotCorpName": "CMPD-0000001-001",
    "lotNumber": 1,
    "parentCorpName": "CMPD-0000001",
    "project": "PROJ-00000001",
    "registrationDate": 1675314000000,
    "supplier": "supplier1",
    "vendorCode": "somecode",
    "vendorName": "vendor1"
  },
  {
    "chemist": "bob",
    "id": 59,
    "lotCorpName": "CMPD-0000002-001",
    "lotNumber": 1,
    "parentCorpName": "CMPD-0000002",
    "project": "PROJ-00000001",
    "registrationDate": 1675314000000,
    "supplier": "supplier2",
    "vendorCode": "somecode2",
    "vendorName": "vendor2"
  }
]
```